eng/mgmt/gulpfile.js

```diff
@@ -151,6 +152,17 @@ var codegen = function(project, cb) {
     return execa(cmd, [], { shell: true, stdio: "inherit" });
 };
 
+var deleteMgmtFolders = function(project) {
+    project = project.split('/')[0]
+    var projectRoot = path.join(sdkRoot, 'sdk', project);
+    fs.readdirSync(projectRoot).forEach(function(file, index) {
+        projectDir = path.join(projectRoot, file)
+        if(fs.lstatSync(projectDir).isDirectory() && file.startsWith('mgmt-')) {
```

Comment:
@weidongxu-microsoft, I'm not a huge fan of startsWith on the directory name to determine management folders. If the management project pom files are all parented correctly they should have a parent artifact of azure-arm-parent. Couldn't we use that to more reliably determine if they're management?

Reply:
Good idea. We should have a "pom.mgmt.xml" in sdk/service root, and I could use that to find all mgmt folders. Let me try it.

I probably will not try with parent project pom. This is JS code (and we actually not want to modify it too much) and I probably do not want to involve too much dependency on XML parsing and POM handling.

Reply:
The POM restructuring that I am doing might end up addressing this ;)